frontend/expr_inferrer.py

```diff
@@ -382,8 +382,30 @@ def infer_func_call(node, context, solver):
 
 
 def infer_attribute(node, context, from_call, solver):
+    """Infer the type of attribute access
+    
+    Cases:
+        - Instance attribute access. ex:
+            class A:
+                def f(self):
+                    pass
+            A().f()
+        - Module access. ex:
+            import A
+            A.f()
+    """
+    # Check if it's a module access
     instance = infer(node.value, context, solver)
+
+    if isinstance(instance, Context):
```

Comment:
The two cases that we don't support:
- Re-assigning the module to a variable.
- Passing the module as a function argument

Added doc for it in `stmt_inference.py` in `_infer_import`